### ISSUE TITLE
How to explicitly set DXF color to black?

### ISSUE THREAD

**Issue by uzername** [ixmilia/dxf#199]
I need to set color of entity to black, but when I use DxfColor.FromIndex(0), I get it as ByBlock. Same happens when I use FromRawValue. I also get ByBlock. It is mentioned here: https://github.com/ixmilia/dxf/issues/97

So, how do I set a color to black? Other colors work, but I'd like black

**Comment by brettfo** [comment]
tl;dr - Black isn't a color that's supported by default.

Long version
====

Colors in DXF files aren't really colors, but they're an index into a palette and each CAD program is free to interpret each index as a specific color as they see fit, or an individual user can even override that, but AutoCAD _seems_ to use the colors that I define [here](https://github.com/ixmilia/dxf/blob/98b1ed65564e4a05a259efcfc5beabe83b7699d3/src/IxMilia.Dxf/DxfColor.cs#L162) with the caveat that you mentioned that index `0` isn't really an index, but a placeholder to mean `BYBLOCK` (`256` = `BYLAYER`, `257` = `BYLAYER`, etc.)

A little further down in issue #97 I mention that starting with AutoCAD 2004 individual entities can have a 24-bit integer representing the RGB color, but that concept doesn't exist on the layer; only on the entity.
